libnetdata/inlined.h

```diff
@@ -31,6 +31,29 @@ static inline uint32_t simple_hash(const char *name) {
     return hval;
 }
 
+/**
+ * Simple N hash
+ *
+ * Calculate the hash for name that has the length len.
+ *
+ * @param name the input to the hash
+ * @param len the length of the input
+ *
+ * @return It returns the hash calculated.
+ */
+static inline uint32_t simple_nhash(const char *name,size_t len) {
+    unsigned char *s = (unsigned char *) name;
+    size_t i;
+    uint32_t hval = 0x811c9dc5;
+    i = 0;
+    do {
```

Comment:
Code style, you're wasting a lot of lines:
```
    i = 0;
    do {
```
 and do while is not readable here.
It should be `for (i=0 ; i < len ; ++i) {`

Reply:
I confess I am working my brain to understand the concept of readable, but I am changing because I know the final result will be the same.